proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -35,6 +35,35 @@ message BreakingChangePolicy {
   BreakingChangeCategory category = 3;
 }
 
+message UniquenessPolicy {
+  optional bool enabled = 1;
+}
+
+message UniquenessState {
+  bool has_collisions = 1;
+  repeated Collision collisions = 2;
+}
+
+message Collision {
+  string id = 1;
+  CollisionType type = 2;
+  string reference = 3;
```

Comment:
What does reference mean here?

Reply:
either the fully qualified name (foo.Bar.Baz)
or proto filepath (a/b/c/v1alpha1/d.proto)